amr-wind/physics/MultiRoughness.H

```diff
@@ -0,0 +1,54 @@
+#ifndef MultiRoughness_H
```

Comment:
Instead of Multi* would it make move sense to call this "Variable*" since it is really about a variable roughness/temp?